### PR TITLE
chore: test package

### PR DESCRIPTION
**Pull request by timsuchanek**

### CODE REVIEW

File: src/packages/cli/src/__tests__/studio.test.ts [hunk 1]
```diff
@@ -6,6 +6,9 @@ import { Studio } from '../Studio'
 
 const STUDIO_TEST_PORT = 5678
 
+// silencium
```
Comment (Jolg42) [inline src/packages/cli/src/__tests__/studio.test.ts:9]: 🤫

File: src/packages/migrate/package.json [hunk 2]
```diff
@@ -10,7 +10,6 @@
     "version": "latest"
   },
   "devDependencies": {
-    "@prisma/client": "workspace:*",
```
Comment (Jolg42) [inline src/packages/migrate/package.json:13]: 🤯 didn't know it was a thing, nice cleanup 😄

File: src/packages/tests/jest.config.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  testMatch: ['**/src/__tests__/**/*.test.ts'],
+  // todo duplicated serializer from client package, should share
```
Comment (Jolg42) [inline src/packages/tests/jest.config.js:5]:
```suggestion
```

File: src/packages/tests/src/__tests__/__helpers__/integrationTest.ts [hunk 4]
```diff
@@ -409,38 +391,20 @@ function getScenarioDir(
...
 /**
  * Get the temporary directory for the scenarios
  */
 function getScenariosDir(databaseName: string, testKind: string) {
-  return Path.join(
-    Path.dirname(pkgDir),
-    'src',
-    '__tests__',
-    'tmp',
+  // use tmp dir instead, as that often times is ramdisk
```
Comment (Jolg42) [inline src/packages/tests/src/__tests__/__helpers__/integrationTest.ts:401]: 🥇 (I'm happy 😄 )

File: src/scripts/ci/publish.ts [hunk 5]
```diff
@@ -939,6 +961,10 @@ async function publishPackages(
     for (const pkgName of currentBatch) {
       const pkg = packages[pkgName]
 
+      if (pkg.name === '@prisma/tests') {
+        continue
```
Comment (Jolg42) [inline src/scripts/ci/publish.ts:965]: I wanted to ask the question "Are we publishing this pkg to npm?" I guess that answers it by a no ✅ Perfect!

**Review by Jolg42 — APPROVED** [review]
Thanks you very much for this PR 💚

**Comment by timsuchanek** [comment]
Thanks! Squashed and merged.

**Pull request merged**